pygmt/src/meca.py

```diff
@@ -293,9 +293,13 @@ def meca(
         newcols = ["longitude", "latitude", "depth"] + param_conventions[convention]
         if "plot_longitude" in spec.columns and "plot_latitude" in spec.columns:
             newcols += ["plot_longitude", "plot_latitude"]
+            spec[["plot_longitude", "plot_latitude"]] = spec[
+                ["plot_longitude", "plot_latitude"]
+            ].astype(str)
```

Comment:
Should we add `str` to the docstring at L192 and L196? Also thinking about DMS coordinates like `23:30:30E` per #647.

Reply:
Done in 42083088.